tests/Entity/UserTest.php

```
@@ -0,0 +1,256 @@
+
+        self::assertFalse($user->isTwoFactorEnabled());
+        self::assertNull($user->getTwoFactorSecret());
+        self::assertNull($user->getTwoFactorRecoveryCodes());
+        self::assertNull($user->getTwoFactorConfirmedAt());
+    }
+
+    public function testUserCanSetupTwoFactor(): void
+    {
+        $user = $this->createValidUser();
+        $secret = 'JBSWY3DPEHPK3PXP';
+        $recoveryCodes = ['code1', 'code2', 'code3', 'code4', 'code5', 'code6', 'code7', 'code8'];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$recoveryCodes'. (undefined)

(UnusedLocalVariable)